tests/rptest/tests/node_id_assignment_test.py

```diff
@@ -119,6 +122,33 @@ def test_rejoin_with_wrong_node_id(self):
                                  expect_fail=True)
 
 
+class NodeIdAssignmentParallel(RedpandaTest):
+    """
+    Test that adds several nodes at once, assigning each a unique node ID.
+    """
+    def __init__(self, test_context):
+        super(NodeIdAssignmentParallel,
+              self).__init__(test_context=test_context, num_brokers=6)
+
+    def setUp(self):
+        self.redpanda.start(self.redpanda.nodes[0:3],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
+
+    @cluster(num_nodes=6)
+    def test_assign_multiple_nodes(self):
+        self.redpanda.start(self.redpanda.nodes[3:],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
```

Comment:
We can save on the number of nodes for this test if it was a sinlge seed node cluster. Number of seeds does not mean much in this test. Alternatively, keeping `num_nodes=6`, it would mean 5 simultaneous joiners which is also better than 3.

Reply:
Done, opted to start with a single node cluster and 3 joiners